src/controllers/beneficial.owner.gov.controller.ts

```diff
@@ -40,7 +40,7 @@ export const getById = (req: Request, res: Response, next: NextFunction) => {
 
     return res.render(BENEFICIAL_OWNER_GOV_PAGE, {
       backLinkUrl: BENEFICIAL_OWNER_TYPE_URL,
-      templateName: BENEFICIAL_OWNER_GOV_PAGE,
+      templateName: `${BENEFICIAL_OWNER_GOV_PAGE}/${id}`,
```

Comment:
Just wondering why this file and the one below have changed here? Surely nothing to do with sign out?

Reply:
`templateName` will be part of the redirect url and needs to be complete with IDs in the getById when MOs or BOs are selected for change/delete. We can have a call

Reply:
But is this change needed (now) for sign out? I can't see how it relates to a requirement in the stories you're working on, but yes, let's have a call if required.

Reply:
templateName is used by the matomo tracking so we don't want to alter the value passed into the template. Perhaps pass id as a separate parameter?

Reply:
Would still like to know which 'sign out' requirement this change relates to.

Reply:
Thanks @markpit, this way will correctly track the real page that the user is selecting, probably I need to discuss it with Sam.

I could concatenate the ID to the template when present as a second implementation option instead of changing the getByID method.

Reply:
Had a talk with Mou and looked at the matomo dashboard and it seems to be able to differentiate between the ids while also putting them all together under managing officer in the tracking. Can't confirm if it's 100% necessary for the 'sign out' functionality but it certainly seems like a useful addition 👍

Reply:
Sounds useful, but if not sign out functionality can we move this change to another story, so that it can be tested against requirements?

Reply:
Just for clarity I posted the snippet related to the sign out link, and you can see the the `href` for the `a` link is constructed using the `templateName`, so we need to have the full and correct path including the ID if you are signing out from MOs or BOs selected. The changes will improve as well the matzo tracking 
```
{% if userEmail %}
   <ul id="navigation">
     <li class="content-email">{{ userEmail }}</li>
     <li>
       <a href="{{ OE_CONFIGS.REGISTER_AN_OVERSEAS_ENTITY_URL }}sign-out?page={{ templateName }}" data-event-id="signout">Sign out</a>
     </li>
   </ul>
   <hr class="govuk-section-break govuk-section-break--visible"> 
{% endif %}
```

Reply:
Right, I'm getting there slowly. Maybe. @mouhajer-ch  Is this needed now (for sign out) because if the user navigates to the 'confirm sign out' page from a BO or MO page, the navigation back to that page - if they abort sign out - won't work without this addition of the Id?